packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/hardened.yaml

```diff
@@ -1,6 +1,6 @@
 {{- $namespaces := dict "_0" .Release.Namespace -}}
-{{- if and .Values.grafana.enabled (or .Values.grafana.sidecar.dashboards.enabled .Values.grafana.defaultDashboardsEnabled) -}}
-{{- $_ := set $namespaces "_1" .Values.grafana.sidecar.dashboards.searchNamespace -}}
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled (not .Values.grafana.defaultDashboards.useExistingNamespace) -}}
```

Comment:
This if statement should be executed if and only if we create a namespace for the default dashboards.

As such, it follow the same logic as the if statement on top of `packages/rancher-monitoring/generated-changes/overlay/templates/grafana/namespaces.yaml`.